### PR TITLE
Issue#299-Solve-the-problem-with-two-not-working-methods-in-ApartmentApiImpl-controller

### PR DESCRIPTION
**Pull request by bbogdasha**
dev
## ZenHub

* [Main ZenHub ticket](https://github.com/ita-social-projects/Home/issues/299)

## Code reviewers

- [ ] @kh0ma 
- [ ] @kizimov 
- [ ] @klaeck
- [ ] @SerhiiPakulia

## Summary of issue

Solve the problem with two not working methods in the ApartmentApiImpl controller

## Summary of change

Added a white list for ApartamentInvitation fields, updated RSQLConfig and created a new converter for it

## Testing approach

Tests are not created. Tests should be written in the following task #300: "Write tests for methods not covered by tests in the ApartmentApiImpl controller".

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by klaeck — APPROVED** [review]

**Review by SerhiiPakulia — APPROVED** [review]

File: home-data-migration/src/main/resources/db/changelog/0.0.1/Issue127.xml [hunk 1]
```diff
@@ -13,6 +13,9 @@
             </column>
             <column name="COOPERATION_NAME" type="VARCHAR(70)">
             </column>
+            <column name="ENABLED" type="BOOLEAN">
```
Comment (kh0ma) [inline home-data-migration/src/main/resources/db/changelog/0.0.1/Issue127.xml]: We cannot change existing migrations. We have to write new one with needed changes. Please read about how liquibase works.

File: home-open-api/src/main/resources/yaml/models/contact.yaml [hunk 2]
```diff
@@ -63,12 +63,12 @@ CreateContact:
       default: false
       type: boolean
       example: false
-  example:
-    {
-      "type": "email",
-      "main": false,
-      "email": "test.put.messages@gmail.com"
-    }
+#  example:
```
Comment (kh0ma) [inline home-open-api/src/main/resources/yaml/models/contact.yaml]: Why we left it commented?

File: home-open-api/src/main/resources/yaml/models/contact.yaml [hunk 3]
```diff
@@ -110,12 +110,12 @@ UpdateContact:
     main:
       type: boolean
       example: false
-  example:
-    {
-      "type": "email",
-      "main": false,
-      "email": "test.put.messages@gmail.com"
-    }
+#  example:
```
Comment (kh0ma) [inline home-open-api/src/main/resources/yaml/models/contact.yaml]: same as above

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/cooperation/invitation/apartment/ApartmentInvitationServiceImpl.java [hunk 4]
```diff
@@ -115,6 +116,7 @@ private void validateSumOwnershipPart(Long apartmentId,
         }
     }
 
+    //enabled
```
Comment (kh0ma) [inline home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/cooperation/invitation/apartment/ApartmentInvitationServiceImpl.java]: why do we need this comment?

**Review by kh0ma — CHANGES_REQUESTED** [review]

File: home-data-migration/src/main/resources/db/changelog/0.0.1/Issue127.xml (hunk 1 above)
Comment (bbogdasha) [reply home-data-migration/src/main/resources/db/changelog/0.0.1/Issue127.xml]: Okay, I got it, I will create a new one.

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/cooperation/invitation/apartment/ApartmentInvitationServiceImpl.java (hunk 4 above)
Comment (bbogdasha) [reply home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/cooperation/invitation/apartment/ApartmentInvitationServiceImpl.java]: Yeah, I forgot about that.

File: home-open-api/src/main/resources/yaml/models/contact.yaml (hunk 2 above)
Comment (bbogdasha) [reply home-open-api/src/main/resources/yaml/models/contact.yaml]: These parameters do not work correctly and do not allow you to use "create" and "update" in contact. We have not found a solution yet.

**Review by kh0ma — APPROVED** [review]

**Pull request merged**